sorc/ncep_post.fd/CMakeLists.txt

```diff
@@ -152,7 +152,6 @@ list(APPEND EXE_SRC
     INITPOST.F
     INITPOST_GFS_NEMS_MPIIO.f
     INITPOST_GFS_NETCDF.f
```

Comment:
@WenMeng-NOAA are you planning to remove INITPOST_GFS_NETCDF.f at some point too?
I know it's good to keep it for a while but am a bit concerned some developers will inadvertently 
work on this subroutine

Reply:
@fossell Is the routine INITPOST_GFS_NETCDF.f  still used for DTC community support?

Reply:
We do not need to support that moving forward. It's already removed from WRFPOST so we've already begun removing that from our testing framework for upcoming UFS releases. I'm ok with with just stating the formats/models we support and leaving it at that, e.g. NetCDF-Para/GFS, NetCDF-Para/LAM, etc.

Reply:
@fossell Thanks for confirming. I will remove it.